.github/workflows/neuron-ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     
     name: ${{ matrix.os }} - ${{ matrix.config.build_mode }} (${{ matrix.config.cmake_option }}${{ matrix.config.config_options }}${{ matrix.config.matrix_eval }})
     
-    timeout-minutes: 30
+    timeout-minutes: 45
```

Comment:
looks like randomly one macos job takes a bit more than 30 min.

Reply:
yeah I have seen the same!